src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -178,16 +178,16 @@ void fillInnerPartsWithIndicatedList(String listName) {
         switch(listName) {
 
         case "blacklist":
-            personListPanel = new PersonListPanel(logic.getFilteredBlacklistedPersonList());
+            personListPanel = new PersonListPanel(logic.getFilteredBlacklistedPersonList(), listName);
             break;
         case "whitelist":
-            personListPanel = new PersonListPanel(logic.getFilteredWhitelistedPersonList());
+            personListPanel = new PersonListPanel(logic.getFilteredWhitelistedPersonList(), listName);
             break;
         case "overduelist":
-            personListPanel = new PersonListPanel(logic.getFilteredOverduePersonList());
+            personListPanel = new PersonListPanel(logic.getFilteredOverduePersonList(), listName);
             break;
         default:
-            personListPanel = new PersonListPanel(logic.getFilteredPersonList());
+            personListPanel = new PersonListPanel(logic.getFilteredPersonList(), "MASTERLIST");
```

Comment:
Use List.Command_Word. "Masterlist" is just the output message we give on the screen. However, internally, the classes use the word "list" to identify it as one of the three lists.